Task_1/fets_challenge/experiment.py

```diff
@@ -454,52 +456,71 @@ def run_challenge_experiment(aggregation_function,
+            binary_dice_wt = get_metric('binary_DICE_WT', round_num, aggregator.tensor_db)
+            binary_dice_et = get_metric('binary_DICE_ET', round_num, aggregator.tensor_db)
+            binary_dice_tc = get_metric('binary_DICE_TC', round_num, aggregator.tensor_db)
+            hausdorff95_wt = get_metric('binary_Hausdorff95_WT', round_num, aggregator.tensor_db)
+            hausdorff95_et = get_metric('binary_Hausdorff95_ET', round_num, aggregator.tensor_db)
+            hausdorff95_tc = get_metric('binary_Hausdorff95_TC', round_num, aggregator.tensor_db)
+
+            # compute the mean dice value
+            round_dice = np.mean([binary_dice_wt, binary_dice_et, binary_dice_tc])
+
+            # update best score
+            if best_dice < round_dice:
+                best_dice = round_dice
+
+            ## CONVERGENCE METRIC COMPUTATION
+            # update the auc score
+            best_dice_over_time_auc += best_dice * round_time
+
+            # project the auc score as remaining time * best dice
+            # this projection assumes that the current best score is carried forward for the entire week
+            projected_auc = (MAX_SIMULATION_TIME - total_simulated_time) * best_dice + best_dice_over_time_auc
+            projected_auc /= MAX_SIMULATION_TIME
+
+            # End of round summary
+            summary = '"**** END OF ROUND {} SUMMARY *****"'.format(round_num)
+            summary += "\n\tSimulation Time: {} minutes".format(round(total_simulated_time / 60, 2))
+            summary += "\n\t(Projected) Convergence Score: {}".format(projected_auc)
+            summary += "\n\tBinary DICE WT: {}".format(binary_dice_wt)
+            summary += "\n\tBinary DICE ET: {}".format(binary_dice_et)
+            summary += "\n\tBinary DICE TC: {}".format(binary_dice_tc)
+            summary += "\n\tHausdorff95 WT: {}".format(hausdorff95_wt)
+            summary += "\n\tHausdorff95 ET: {}".format(hausdorff95_et)
+            summary += "\n\tHausdorff95 TC: {}".format(hausdorff95_tc)
+
+            experiment_results['round'].append(round_num)
+            experiment_results['time'].append(total_simulated_time)
+            experiment_results['convergence_score'].append(projected_auc)
+            experiment_results['binary_dice_wt'].append(binary_dice_wt)
+            experiment_results['binary_dice_et'].append(binary_dice_et)
+            experiment_results['binary_dice_tc'].append(binary_dice_tc)
+            experiment_results['hausdorff95_wt'].append(hausdorff95_wt)
+            experiment_results['hausdorff95_et'].append(hausdorff95_et)
+            experiment_results['hausdorff95_tc'].append(hausdorff95_tc)
+
+            logger.info(summary)
+        else:
+            # update the auc score
+            best_dice_over_time_auc += best_dice * round_time
+            summary = f'Skipped challenge validation metrics for round {round_num}'
+            logger.info(summary)
+
+        if save_checkpoints:
```

Comment:
Looks like you execute the block to save checkpoints, two times (one over-writing the other)?

Reply:
Thanks @brandon-edwards  for catching this! This was a merge error. Fixing now